packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -158,22 +158,22 @@ export async function requestFunding(
     body: JSON.stringify(postBody),
   })
 
-  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions -- It's a FaucetWallet
-  const body = (await response.json()) as FaucetWallet
-  // "application/json; charset=utf-8"
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- it can be anything
```

Comment:
Confused what you mean

Reply:
It seems fetch and fetch.json don't have template types :(

Would it be easier to just assert the type?

Reply:
The fetch api only lets you retrieve the body once.  At this point it could be an error and not guaranteed to be a FaucetWallet.  That is why the casting was moved lower.